source/about/install-mattermost-app-in-microsoft-teams.rst

```diff
@@ -4,13 +4,15 @@ Install the Mattermost App in Microsoft Teams
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-.. note::
+With `Mattermost for Microsoft Teams </about/mattermost-for-microsoft-teams.html>`__, you can embed your Mattermost workspace within your Microsoft Teams instance and take advantage of `embedded app features <#benefits-of-the-embedded-app>`_.
 
-    You must `install the Microsoft Teams plugin in Mattermost </plugins/install-mattermost-for-microsoft-teams-plugin>`__ before you can install the Mattermost app in Microsoft Teams. 
+.. important::
+
+    You must `install the Microsoft Teams plugin in Mattermost </plugins/install-mattermost-for-microsoft-teams-plugin>`__ before you can embed the Mattermost app within Microsoft Teams.
 
 To install the Mattermost app in Microsoft Teams:
 
-1. In Mattermost, as an administrator, go to System Console > Plugins > Mattermost for Microsoft Teams. At the bottom of the settings page, select **Download Manifest** to download a ZIP file containing the Microsoft Teams application manifest pre-configured for your Mattermost workspace.
+1. In Mattermost, as an administrator, go to **System Console > Plugins > Mattermost for Microsoft Teams**. At the bottom of the settings page, select **Download Manifest** to download a ZIP file containing the Microsoft Teams application manifest pre-configured for your Mattermost workspace.
```

Comment:
@wiggin77 - Is this a customer-specific step or something generic that applies to all customers interested in embedding the Mattermost app in MS Teams?

Reply:
This step is generic- applies to anyone wanting to embed.

Reply:
Thanks for confirming, @wiggin77. I don't see this System Console UI in the [MS Teams test instance](https://msteams-test.test.mattermost.cloud/admin_console/plugins/plugin_com.mattermost.msteams-sync). Is that intentional? Would I need a newer build perhaps?

Reply:
@jasonblais - Is the System Console plugin setting going to remain as **MSTeams Sync**, or will it be renamed?

Reply:
@cwarnermm 

> I don't see this System Console UI in the [MS Teams test instance](https://msteams-test.test.mattermost.cloud/admin_console/plugins/plugin_com.mattermost.msteams-sync). Is that intentional? Would I need a newer build perhaps?

It will require a newer version (v0.4). Doug will be cutting it this afternoon, after which the MS Teams test instance can be updated with it.

> Is the System Console plugin setting going to remain as MSTeams Sync, or will it be renamed?

There is a proposal to rename the plugin to be more generic from `MSTeams Sync`:
https://github.com/mattermost/mattermost-plugin-msteams-sync/pull/210

Reply:
Update - it may not be Doug creating, it may be BrightScout. But regardless, a new version of the plugin is being cut after which the test instance can be updated

Reply:
Thanks, @jasonblais!

Reply:
> I don't see this System Console UI in the [MS Teams test instance](https://msteams-test.test.mattermost.cloud/admin_console/plugins/plugin_com.mattermost.msteams-sync).

A new version of MM server will need to be deployed there, plus the latest plugin that will be built.

Reply:
> It will require a newer version (v0.4).

@jasonblais I kicked off the release before I saw this.  It is v.0.3.9.

Reply:
Sounds good, noted

@cwarnermm Doug made a good point - we'll need to update the test instance to v8.0.1 which is cut around Wednesday-Thursday of this week, and then update the plugin to v0.3.9: https://github.com/mattermost/mattermost-plugin-msteams-sync/releases/tag/v0.3.9

Those are the minimum compatible versions for the app,

Reply:
Thanks both! https://github.com/mattermost/docs/pull/6540 documents the plugin version.